public/templates/header.php

```diff
@@ -5,47 +5,30 @@
     <meta http-equiv="x-ua-compatible" content="ie=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-    <script type="text/javascript" src="https://code.jquery.com/jquery-3.6.0.slim.min.js" integrity="sha256-u7e5khyithlIdTpu22PHhENmPcRdFiHRjhAuHcs05RI=" crossorigin="anonymous"></script>
```

Comment:
we should retain this for now, some of the functionality that we have depends on this. Until we rewrite some of the code this dependency should stay